### DISCUSSION TITLE
Not able to verify via upload of image of education registration documents

### DISCUSSION THREAD

**Discussion by hassanadnan** [community/community#38028] [GitHub Education]
Hi, I am in Linkoping University Sweden, I have tried to verify my document but it is not working. Anyone else faced same issue? Is it may be because the name on the document is in Swedish (the only difference is because of the two dots over o)

**Comment by jquanC** [comment]
It may take time for the verification to succeed, or does it have a reason for not passing?

**Reply by hassanadnan** [reply]
Actually, the automatic page is not verifying it. So it is not even being processed.

**Reply by jquanC** [reply]
I think you could open the console and try again to see if it's a network issue or something.

**Comment by hassanadnan** [comment]
To take the benefits of the github education account.

**Comment by lauralimatoloni** [comment]
hello, I'm having the same problem, I've been trying for a few days, I upload the documentation but when I click on the send button, a circle appears loading and it never loads, I've waited for almost 1h thinking it's processing, it doesn't load, don't proceed with the page to submit the documentation.

**Comment by samus-aran** [comment]
Hi @hassanadnan  & @lauralimatoloni 👋 

There was previously a glitch with uploading large images. But that is resolved. How do you two get on with re-applying?
